lib/components/DescriptionWithAction.tsx

```diff
@@ -0,0 +1,61 @@
+import type { ReactNode } from 'react'
+import { Tag } from 'lucide-react'
+import { tx, tw } from '../twind'
+import { noop } from '../util/noop'
+import { Span } from './Span'
+import { Button } from './Button'
+
+export type DescriptionWithActionProps = {
+  title: string,
+  description: string,
+  leadingIcon?: 'none' | 'label',
```

Comment:
Any reason why we should not pass ReactNodes? Can we strictly type "Icons"?

Reply:
Because it is easier to manage for now to align a fixed icon with the text, additionally the storybooks don't work with ReactNode, because tailwind styles are not applied for the ReactNode parameters.

We could use a example component which probably makes it compatible but introduces a new file.

Suggestions for which solution is preferred? @MaxSchaefer

Reply:
Okay, check! Fine with that.